src/assets/js/index.js

```diff
@@ -9,14 +9,10 @@ document.addEventListener("DOMContentLoaded", () => {
             document.body.classList.toggle('mobile-nav-opened');
         });
     }
-
-    const extLinks = document.querySelectorAll("a");
-    [].forEach.call(extLinks, (a) => {
-        const link = a.getAttribute("href");
-        if (link.indexOf("http") === 0) {
-            a.setAttribute("target", "_blank");
-            a.setAttribute("rel", "noopener");
-        }
+    const langLinks = document.querySelectorAll(".c-lang a");
```

Comment:
Instead of listening to link clicks, we may parse url when user directly visits `radzima.app/ru`. We may extract locale from path. The benefit of this approach would be that not only link clicks will set preferred locale in localStorage, but any visit to direct url.